### ISSUE TITLE
Clickable Spotify icon

### ISSUE THREAD

**Issue by balazsorban44** [panz3r/hyper-spotify#26]
I looked at your source code, and I suppose you intend to turn the spotify icon into a clickable icon in the future, that opens spotify. Is there anything holding you back to implement this right away?

https://github.com/panz3r/hyper-spotify/blob/cfb41050f5a58d831ffafd93e68959ec7a95ea49/src/containers/HyperSpotifyWidget.js#L216-L222

**Comment by panz3r** [comment]
Hi @balazsorban44 ,

Yes, that was an old idea that I had.
I might actually implement it but I'd need someone to test it out (especially on Linux).

**Comment by panz3r** [comment]
Hi @balazsorban44 ,

If you'd like to try this feature, I just released version `v5.1.0-beta.1` that you can install by editing you're `.hyper.js` as follow
```js
module.exports = {
  ...
  plugins: ['hyper-spotify#5.1.0-beta.1']
  ...
}
```

Please let me know if it's working.

**Comment by balazsorban44** [comment]
I had to manually install it by 
```sh
cd .hyper_plugins && npm install
```
but yes, it looks like it's working! It logs "Start Spotify" in the dev console though. Maybe it's not necessary.

**Comment by panz3r** [comment]
@balazsorban44 ,

Just published `v5.1.0-beta.2` which removes the log message (which actually was useless).
I'm planning to release the stable version as soon as possible, thanks again for testing this out on Linux.

**Comment by panz3r** [comment]
Released `v5.1.0`
